shared/src/main/java/com/sap/psr/vulas/shared/json/model/ExemptionScope.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * This file is part of Eclipse Steady.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+package com.sap.psr.vulas.shared.json.model;
+
+import java.util.Map;
+
+import org.apache.commons.configuration.Configuration;
+
+import com.sap.psr.vulas.shared.enums.Scope;
+
+public class ExemptionScope implements IExemption {
+
+	/**
+	 * Deprecated configuration setting <code>REP_EXCL_UNASS="vulas.report.exceptionScopeBlacklist"</code>.
+	 */
+	public static final String DEPRECATED_CFG = "vulas.report.exceptionScopeBlacklist";
+
+	/**
+	 * New configuration setting <code>REP_EXCL_UNASS="vulas.report.exemptScope"</code>.
+	 */
+	public static final String CFG = "vulas.report.exemptScope";
```

Comment:
"vulas." should be removed to keep on using configurations without the vulas prefix as it was done until now